app/controllers/users_controller.rb

```diff
@@ -0,0 +1,59 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+    @user.build_address
+  end
+  
+  def create
+    @user = User.new(user_params)
+    # @user.build_address    
```

Comment:
Remove commented code from all the files

Reply:
Okay